client/src/App.js

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import logo from "./logo.svg";
 import "./style.css";
-
```

Comment:
To avoid changes, there should not be any unnecessary changes in the App.js file.